packages/headless/src/features/generated-answer/generated-answer-analytics-actions.ts

```diff
@@ -32,7 +38,7 @@ export const logOpenGeneratedAnswerSource = (
       return client.makeOpenGeneratedAnswerSource({
         generativeQuestionAnsweringId,
         permanentId: citation.permanentid,
-        id: citation.id,
```

Comment:
This is an undocumented breaking change in cajs done in a patch version, be careful about those.

Reply:
Indeed this is a breaking change that we introduced this morning in cajs after discussion with multiple parts,   the PR can be found here: https://github.com/coveo/coveo.analytics.js/pull/438#discussion_r1348801565

After these discussions, and since the RGA feature is still in a beta state, the decision was to change the name of this parameter now.